### PR TITLE
[#87] Add accept and reject challenge commands

### PR DESCRIPTION
**Pull request by allenkinzalow**
**Description**

- Add `accept_challenge` and `reject_challenge` commands.
- Remove all `await`'s from interaction responses. The bot should not halt/wait for an individual command process to respond.

### CODE REVIEW

File: src/discord/actions/approveChallenge.ts
Comment (Hammer-God) [inline src/discord/actions/approveChallenge.ts:13]: There should be a check to ensure that the Challenge Card status is in the Approval state before we can update the challenge card to be approved or rejected. The same applies for the rejectChallenge.

File: src/discord/actions/approveChallenge.ts
Comment (Hammer-God) [inline src/discord/actions/approveChallenge.ts:1]: This is dangerous code. I could look at evidence, approve it without realizing that maybe somebody else already approved it and I might have then accidentally approved the next tier of card up if they completed those challenges in the meantime. We should explicitly check the difficulty (maybe include it as a parameter in the commands) and approval state of that challenge card that we are looking at.

Comment (allenkinzalow) [reply src/discord/actions/approveChallenge.ts:1]: I've removed the approval and rejection buttons. The approval and rejection commands now have a required `difficulty` param and also look up the challenge card by pending status. If you can, give this a test before approving.

**Review by Hammer-God — APPROVED** [review]

**Pull request merged**
